client/src/components/landing/landing.css

```diff
@@ -31,9 +31,10 @@
   font-family: 'Lato', sans-serif;
 }
 
-p.caption {
+figcaption.caption {
   margin: 10px 0 0;
-  font-size: 0.8rem;
+  font-weight: 600;
+  text-align: center;
   color: var(--quaternary-color);
 }
```

Comment:
Btw I am happy to revert this, it's outside the scope of the PR, and it isn't important.